warehouse/jobs/handlers.go

```diff
@@ -106,53 +101,39 @@ func (a *AsyncJobWh) AddWarehouseJobHandler(w http.ResponseWriter, r *http.Reque
 }
 
 func (a *AsyncJobWh) StatusWarehouseJobHandler(w http.ResponseWriter, r *http.Request) {
-	if r.Method == http.MethodGet {
-		a.logger.Info("Got Async Job Status Request")
-		a.logger.LogRequest(r)
-		jobRunId := r.URL.Query().Get("job_run_id")
-		taskRunId := r.URL.Query().Get("task_run_id")
-
-		sourceId := r.URL.Query().Get("source_id")
-		destinationId := r.URL.Query().Get("destination_id")
-		workspaceId := r.URL.Query().Get("workspace_id")
-		payload := StartJobReqPayload{
-			TaskRunID:     taskRunId,
-			JobRunID:      jobRunId,
-			SourceID:      sourceId,
-			DestinationID: destinationId,
-			WorkspaceID:   workspaceId,
-		}
-		if !validatePayload(payload) {
-
-			a.logger.Errorf("[WH]: Error Invalid Status Parameters")
-			http.Error(w, "invalid request", http.StatusBadRequest)
-			return
-		}
-		startJobPayload := StartJobReqPayload{
-			JobRunID:      jobRunId,
-			TaskRunID:     taskRunId,
-			SourceID:      sourceId,
-			DestinationID: destinationId,
-		}
-		a.logger.Infof("Got Payload job_run_id %s, task_run_id %s \n", startJobPayload.JobRunID, startJobPayload.TaskRunID)
+	a.logger.Info("[WH-Status Handler] Got Async Job Status Request")
+	if !a.enabled {
+		a.logger.Errorf("[WH]: Error Warehouse Jobs API not initialized")
+		http.Error(w, "warehouse jobs api not initialized", http.StatusBadRequest)
+		return
+	}
+	a.logger.LogRequest(r)
+	jobRunId := r.URL.Query().Get("job_run_id")
+	taskRunId := r.URL.Query().Get("task_run_id")
 
-		if !a.enabled {
-			a.logger.Errorf("[WH]: Error Warehouse Jobs API not initialized")
-			http.Error(w, "warehouse jobs api not initialized", http.StatusBadRequest)
-			return
-		}
+	sourceId := r.URL.Query().Get("source_id")
+	destinationId := r.URL.Query().Get("destination_id")
+	workspaceId := r.URL.Query().Get("workspace_id")
+	payload := StartJobReqPayload{
+		TaskRunID:     taskRunId,
+		JobRunID:      jobRunId,
+		SourceID:      sourceId,
+		DestinationID: destinationId,
+		WorkspaceID:   workspaceId,
+	}
+	if !validatePayload(payload) {
+		a.logger.Errorf("[WH]: Error Invalid Status Parameters")
```

Comment:
Why "invalid status parameters"? Isn't "invalid payload" a better fit in this case?